app/src/main/java/com/canopas/yourspace/ui/flow/settings/space/SpaceProfileViewModel.kt

```
@@ -93,6 +95,48 @@ class SpaceProfileViewModel @Inject constructor(
+        val currentTimeMillis = System.currentTimeMillis()
+        val expireTimeMillis = createTimeMillis + durationMillis
+        val diffInMillis = expireTimeMillis - currentTimeMillis
+
+        if (diffInMillis <= 0) {
+            Timber.e("XXX :- The invite code is expired.")
+            return "Expired"
+        }
+
+        val hours = TimeUnit.MILLISECONDS.toHours(diffInMillis) % 48
+        val minutes = TimeUnit.MILLISECONDS.toMinutes(diffInMillis) % 60
+        return String.format("%02d hours %02d minutes", hours, minutes)
```

detekt (1.23.7)

[warning] String.format("%02d hours %02d minutes", hours, minutes) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)